src/main/java/io/github/jav/exposerversdk/ExpoPushTicket.java

```diff
@@ -17,6 +17,8 @@
 
     @JsonProperty("id")
     public String id = null;
+    @JsonProperty("to")
```

Comment:
@celandro 
AFAICT, the 'to' field _must_ not be null.
Do you see any scenarios where it should be allowed to be NULL?

Reply:
It was already set to null in the prior code. I did not change it. I believe it should always be set.  If we wanted to be more specific we should change everything to use Optional and @Notnull annotations but I would argue that should be a separate PR, this one is already too big